components/AppBar/src/MDCAppBarViewController.m

```diff
@@ -276,6 +288,15 @@ - (void)dismissSelf {
   }
 }
 
+#pragma mark - Private
+
+- (void)adjustHeightBasedOnHeaderStackView {
+  CGFloat heightSum = 0;
+  heightSum += [self.headerStackView.topBar sizeThatFits:self.view.frame.size].height;
+  heightSum += [self.headerStackView.bottomBar sizeThatFits:self.view.frame.size].height;
```

Comment:
Always use `self.view.bounds` when calculating dimensions of subviews.

```objc
  heightSum += [self.headerStackView.topBar sizeThatFits:self.view.bounds.size].height;
  heightSum += [self.headerStackView.bottomBar sizeThatFits:self.view.bounds.size].height;
```

Reply:
Addressed in 1b97d553ff0a58d5597bbe7382a338085125677f.